logbook-okhttp/src/main/java/org/zalando/logbook/okhttp/RemoteResponse.java

```diff
@@ -0,0 +1,93 @@
+package org.zalando.logbook.okhttp;
+
+import okhttp3.MediaType;
+import okhttp3.Response;
+import okhttp3.ResponseBody;
+import org.zalando.logbook.HttpResponse;
+import org.zalando.logbook.Origin;
+import org.zalando.logbook.RawHttpResponse;
+
+import java.io.IOException;
+import java.nio.charset.Charset;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+
+import static java.nio.charset.StandardCharsets.UTF_8;
+import static java.util.Objects.requireNonNull;
+import static okhttp3.ResponseBody.create;
+
+final class RemoteResponse implements RawHttpResponse, HttpResponse {
+
+    private Response response;
+    private byte[] body;
+
+    public RemoteResponse(Response response) {
+        this.response = response;
+    }
+
+    @Override
+    public int getStatus() {
+        return response.code();
+    }
+
+    @Override
+    public String getProtocolVersion() {
+        // TODO find the real thing
+        return "HTTP/1.1";
```

Comment:
`return response.protocol().toString();` (+ to upper case?)

https://github.com/square/okhttp/blob/f23067f6ccba94db598485530d722fcb60f9e3e0/okhttp/src/main/java/okhttp3/Protocol.java#L41